README.md

```diff
@@ -1 +1,42 @@
 # DabApps ESLint Configuration
+
+## Install
+
+If there's a suitable version of eslint-config, include the eslint-config in your package.json.
+Recommended to pin a specific version.
+
+```json
+"devDependencies": {
+  "eslint-config": "git://github.com/dabapps/eslint-config.git#1.0.0",
+}
+```
+
+If you need additional plugins or some slightly different config, you can just copy the rules & create your own eslintrc file (but this shouldn't be nessecary).
+
+You'll also need to include eslint & any other plugins that are needed in your package.json.
+You can find out suitable versions from eslint-config's package.json.
+
+```json
+"devDependencies": {
+  "eslint": "=1.5.1",
```

Comment:
If the eslint package is included in the configs package.json as a full dependancy, then we don't need to include it here.